cypress/integration/BonnieFHIR/WebInterface/Dashboard/UploadDialog/ErrorHandling.spec.js

```diff
@@ -25,7 +25,8 @@ describe('Dashboard: Upload Dialog: Error handling', () => {
         //verify the error message
 
         cy.get(importMeasureDialog.errorDialog).invoke('text').then((text) => {
-            expect(text).to.include('The uploaded measure bundle does not contain the proper FHIR JSON file.' +
+            expect(text).to.include('The uploaded file is not a valid Measure Authoring Tool (MAT) export of a FHIR Based Measure.')
```

Comment:
The error text has changed because Bonnie is looking for just 1 json file in a bundle. But a QDM package has multiple json files.